.github/workflows/ci.yml

```diff
@@ -11,6 +11,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+
+      - name: Dependabot metadata
+        if: github.actor == 'dependabot[bot]' && github.event_name == 'pull_request'
+        id: dependabot-metadata
+        uses: dependabot/fetch-metadata@v1.3.1
```

Comment:
use the major version specifier only please, as we do with all actions, to avoid too much noise from dependabot when trying to bump this version